x/erc20/keeper/token_pairs.go

```diff
@@ -143,10 +145,50 @@ func (k Keeper) IsDenomRegistered(ctx sdk.Context, denom string) bool {
 	return store.Has([]byte(denom))
 }
 
+// GetCoinAddress returns the corresponding ERC-20 contract address for the
+// given denom.
+// If the denom is not registered and its an IBC voucher, it returns the address
+// from the hash of the ICS20's DenomTrace Path.
+func (k Keeper) GetCoinAddress(ctx sdk.Context, denom string) (contractAddress common.Address, ok bool) {
```

Comment:
neither `contractAddress` nor `ok` are ever used, so no need to specify them in the signature
```suggestion
func (k Keeper) GetCoinAddress(ctx sdk.Context, denom string) (common.Address, bool) {
```